### PR TITLE
eospac: initial commit

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

File: var/spack/repos/builtin/packages/eospac/package.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from spack import *
+
+
+class Eospac(Package):
+    """A collection of C routines that can be used to access the Sesame data
+       library.
+    """
+
+    homepage = "https://laws.lanl.gov/projects/data/eos.html"
+    url      = "https://laws.lanl.gov/projects/data/eos/get_file.php?package=eospac&filename=eospac_v6.4.0beta.1_r20171213193219.tgz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/eospac/package.py]: Can you add the following list url?
```python
list_url = "https://laws.lanl.gov/projects/data/eos/eospacReleases.php"
```
This allows things like `spack versions` and `spack checksum` to pick up new versions.

File: var/spack/repos/builtin/packages/eospac/package.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    version('6.4.0beta.1', 'e4e4beabf946f0b8953532832002afc2')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/eospac/package.py]: We normally don't add alpha/beta versions of packages unless we have to. Can you add the latest stable release instead? If you really want the beta version, the stable version should be set to `preferred=True`.

File: var/spack/repos/builtin/packages/eospac/package.py (hunk 1 above)
Comment (junghans) [reply var/spack/repos/builtin/packages/eospac/package.py]: Done

File: var/spack/repos/builtin/packages/eospac/package.py (hunk 2 above)
Comment (junghans) [reply var/spack/repos/builtin/packages/eospac/package.py]: Done

**Pull request merged**